packages/nx-plugin/generators.json

```diff
@@ -4,6 +4,11 @@
       "factory": "./src/generators/init/generator",
       "schema": "./src/generators/init/schema.json",
       "description": "init generator"
+    },
+    "add-to-project": {
```

Comment:
Maybe we can get inspired by existing generator names from Nx.

Here the vite generators: https://github.com/nrwl/nx/blob/master/packages/vite/generators.json